contracts/token/ERC1155/utils/ERC1155Receiver.sol

```diff
@@ -7,7 +7,9 @@ import {IERC1155Receiver} from "../IERC1155Receiver.sol";
 import {IERC165, ERC165} from "../../../utils/introspection/ERC165.sol";
 
 /**
- * @dev _Available since v3.1._
+ * @dev Basic contract implementing the ERC-165 interface for {IERC1155Receiver}.
+ *
+ * NOTE: This contract does not suffice to receive tokens. See {ERC1155Holder}.
  */
 abstract contract ERC1155Receiver is ERC165, IERC1155Receiver {
```

Comment:
I don't really understand why we have separate ERC1155Receiver and ERC1155Holder contracts. I think we should merge them, and only keep ERC1155Holder.